sklearn/ensemble/_stacking.py

```diff
@@ -0,0 +1,576 @@
+        # Remove the None from the method as well.
+        self.predict_method_ = [
+            meth for (meth, est) in zip(self.predict_method_, estimators_)
+            if est not in (None, 'drop')
+        ]
+
+        X_meta = self._concatenate_predictions(X, X_meta)
+        self.final_estimator_.fit(X_meta, y)
+
+        return self
+
+    def transform(self, X):
+        """Return class labels or probabilities for X for each estimator.
+
+        Parameters
+        ----------
+        X : {array-like, sparse matrix}, shape (n_samples, n_features)
+            Training vectors, where n_samples is the number of samples and
+            n_features is the number of features.
+
+        Returns
+        -------
+        y_preds : ndarray, shape (n_samples, n_estimators)
+            Prediction outputs for each estimator.
+        """
+        check_is_fitted(self, 'estimators_')
+        return self._concatenate_predictions(
+            X, [getattr(est, meth)(X)
+                for est, meth in zip(self.estimators_, self.predict_method_)
+                if est not in (None, 'drop')]
+        )
+
+    @if_delegate_has_method(delegate='final_estimator_')
+    def predict(self, X, **predict_params):
+        """Predict target for X.
+
+        Parameters
+        ----------
+        X : {array-like, sparse matrix}, shape (n_samples, n_features)
+            Training vectors, where n_samples is the number of samples and
+            n_features is the number of features.
+
+        **predict_params : dict of string -> object
+            Parameters to the `predict` called by the `final_estimator`. Note
+            that this may be used to return uncertainties from some estimators
+            with `return_std` or `return_cov`.
+
+        Returns
+        -------
+        y_pred : ndarray, shape (n_samples,) or (n_samples, n_output)
+            Predicted targets.
+        """
+
+        check_is_fitted(self, ['estimators_', 'final_estimator_'])
+        return self.final_estimator_.predict(
+            self.transform(X), **predict_params
+        )
+
+
+class StackingClassifier(_BaseStacking, ClassifierMixin):
+    """Stacked of estimators using a final classifier.
+
+    Stacked generalization consists in stacking the output of individual
+    estimator and use a classifier to compute the final prediction. Stacking
+    allows to use the strength of each individual estimator by using their
+    output as input of a final estimator.
+
+    Note that `estimators` are fitted on the full `X` while `final_estimator`
+    is trained using cross-validated predictions of the base estimators using
+    `cross_val_predict`).
+
+    .. versionadded:: 0.22
+
+    Read more in the :ref:`User Guide <stacking>`.
+
+    Parameters
+    ----------
+    estimators : list of (string, estimator) tuples
+        Base estimators which will be stacked together. An estimator can be set
+        to None or 'drop' using `set_params`.
```

Comment:
We seem to be move away from using `None`.

Reply:
I think that we support both None and drop in fact.